docs/source/additional/privacy-strategy.rst

```diff
@@ -0,0 +1,133 @@
+How Substra thinks about privacy.
+=================================
+
+.. _Privacy Strategy:
+
+As proponents of Privacy Enhancing Technologies (PETs), the maintainers of Substra believe it is essential for us to outline our stance on these technologies and provide a comprehensive view of our current understanding of the field.
+
+Here we elaborate on what PETs are and provide a quick summary of the most popular ones. We then take a look at the different types of attacks that can result in data leakage or data theft and then also propose best practices for project governance and security in order to mitigate the risks of potential attacks. We also elaborate on the collaboration required between different personas to ensure data integrity and safe handling of data. 
+
+As pointed out by Katharine Jarmul in her (primer on Privacy Enhancing Technologies)[https://martinfowler.com/articles/intro-pet.html], “Privacy is a technical, legal, political, social and individual concept.” As such, technical solutions are an important part of the answer, but they must be used in conjunction with good data governance and reliable security measures.
+
+This part of the documentation will be updated regularly to reflect the latest available research.
+
+Privacy Enhancing Technologies.
+-------------------------------
+
+We touch on a **few** of the main technologies that are making collaborative data sharing possible today in ways that can be considered more secure.
+
+**Federated Learning**:
+FL allows Machine Learning models to be sent to servers where they can train and test on data without having to ever move the data from its original location. This idea however is not restricted to machine learning and is then referred to as Federated Analytics. The Substra software allows for both Federated Learning and Federated Analytics.
+
+**Secure Enclaves**: 
+These are hardware-based features that provide an isolated environment to store and process data. A secure enclave is essentially an ultra secure space within a larger secure system. Although they are excellent for safely storing data, the privacy is hardware dependent and places trust in a physical chip as opposed to encryption.
+
+**Differential Privacy**:
+This approach adds noise throughout processing to prevent inferring item-specific information. This comes with a tradeoff between privacy and performance though, as the addition of noise might cause models to be less accurate. DP provides theoretical bounds on how hard it is to infer the presence of a given individual in a dataset when it is used.
```

Comment:
```suggestion
This approach adds noise throughout processing to prevent inferring item-specific information. This comes with a tradeoff between privacy and performance though, as the addition of noise might cause models to be less accurate. Differential Privacy provides theoretical bounds on how hard it is to infer the presence of a given individual in a dataset when it is used.
```